documentation/domains/Domain.json

```diff
@@ -576,7 +605,7 @@
           "type": "string"
         },
         "serverPod": {
-          "description": "Customization affecting the generation of Pods for WebLogic Server instances.",
+          "description": "Customization affecting the generation of Pods for WebLogic Server instances or Introspector Job Pod.",
```

Comment:
I don't think the 'ManagedServer' scope can have an affect on the introspector pod, which I think only historically pays attention to 'adminServer' and 'domain' level scopes.

Reply:
The serverPod is shared between domain, cluster and server scopes. Its currently not possible to give different description at different scopes.